python/timeplus/dbapi.py

```diff
@@ -84,16 +85,24 @@ def connect(
     Connection: A Connection object.
     """
     address = f"{scheme}://{host}:{port}"
-    apikey = password
     workspace = path
-    return Connection(address, apikey, workspace)
+    if user is not None and password is not None:
```

Comment:
Proton doesn't require password so, password can be `None` here.

Reply:
is that empty not None?

Reply:
I assum in case the password in None, it equals no authentication?